src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
  * Possibility to use all `Ip` constraint versions for `Cidr` constraint
  * Add `list` and `associative_array` types to `Type` constraint
  * Add the `Charset` constraint
+ * Add the `Enum` constraint
```

Comment:
Needs to be adjusted to the new name of the constraint. Same goes for the PR title.

Reply:
Sure, I updated the PR with some feedbacks but not finished yet. I will fix it